components/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider.jsp

```diff
@@ -1235,7 +1237,7 @@
                                     <input type="checkbox"
                                                            name="enableSingleLogout" value="true"
                                                            onclick="disableLogoutUrl(this);"
-                                            <%=(isEditSP && provider.getDoSingleLogout()) ? "checked=\"checked\"" : ""%>/>
+                                            <%= isSingleLogoutEnabled(isEditSP, provider) ? "checked" : ""%>/>
```

Comment:
shall we replace inline if with an if else clause?

Reply:
Responded in previous comment.